### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'FIRE_MODES')

### ISSUE THREAD

**Issue by QuentinGruber** [QuentinGruber/h1z1-server#1111]
```
 at /usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3204:117
    at Array.map (<anonymous>)
    at LiveZoneServer.pGetRemoteWeaponData (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3203:36)
    at /usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3246:29
    at Array.forEach (<anonymous>)
    at LiveZoneServer.pGetRemoteWeaponsData (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3242:43)
    at LiveZoneServer.lootItem (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3960:45)
    at LiveZoneServer.pickupItem (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3982:14)
    at zonePacketHandlers.commandPlayerSelect (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zonepackethandlers.js:585:28)
    at zonePacketHandlers.processPacket (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zonepackethandlers.js:1838:22)
```
```
An error occurred while processing a packet :  {
  name: 'Command.PlayerSelect',
  data: { characterId: '0xc869189c095a4aeb', guid: '0xa91405e3ecd77cd3' }
}
```

**Comment by QuentinGruber** [comment]
also
```
An error occurred while processing a packet :  {
  name: 'Character.FullCharacterDataRequest',
  data: { characterId: '0x1a450355d22aee40' }
}
```

**Comment by QuentinGruber** [comment]
Having better types for this could lead us to discover how the error happen

**Comment by QuentinGruber** [comment]
This is what disable all equipment and stuff overtime on main servers

**Comment by QuentinGruber** [comment]
TypeError: Cannot read properties of undefined (reading 'map') even with the dirty fix of [8ab9200](https://github.com/QuentinGruber/h1z1-server/commit/8ab92004d76fda8ec3540bf91df38be7d956dbbd) it still happens but equipements isn't fucked up
